### PR TITLE
Snap Kit SDK 1.6.8 Update

### PR DESCRIPTION
**Pull request by bkirbysc**
This PR adds the files for the Snap Kit SDK 1.6.8 update

### CODE REVIEW

**Review by driefler — APPROVED** [review]

**Review by kemooo3333 — COMMENTED** [review]
H

**Review by kemooo3333 — COMMENTED** [review]
‏rsh-43

**Comment by kemooo3333** [comment]
> ‏rsh-43

Password

File: SCSDKBitmojiKit.json [hunk 1]
```diff
@@ -1 +1 @@
...
...it-carthage/repo/SCSDKBitmojiKit/1.4.3/SCSDKBitmojiKit-1.4.3.zip", "1.4.4": "https://raw.githubusercontent.com/Snapchat/snap-kit-carthage/repo/SCSDKBitmojiKit/1.4.4/SCSDKBitmojiKit-1.4.4.zip", "1.5.0": "https://raw.githubusercontent.com/Snapchat/snap-kit-carthage/repo/SCSDKBitmojiKit/1.5.0/SCSDKBitmojiKit-1.5.0.zip", "1.3.3": "https://raw.githubusercontent.com/Snapchat/snap-kit-carthage/repo/SCSDKBitmojiKit/1.3.3/SCSDKBitmojiKit-1.3.3.zip", "1.3.2": "https://raw.githubusercontent.com/Snapchat/snap-kit-carthage/repo/SCSDKBitmojiKit/1.3.2/SCSDKBitmojiKit-1.3.2.zip", "1.3.1": "https://raw.githubusercontent.com/Snapchat/snap-kit-carthage/repo/SCSDKBitmojiKit/1.3.1/SCSDKBitmojiKit-1.3.1.zip", "1.3.0": "https://raw.githubusercontent.com/Snapchat/snap-kit-carthage/repo/SCSDKBitmojiKit/1.3.0/SCSDKBitmojiKit-1.3.0.zip", "1.2.2": "https://raw.githubusercontent.com/Snapchat/snap-kit-carthage/repo/SCSDKBitmojiKit/1.2.2/SCSDKBitmojiKit-1.2.2.zip", "1.3.5": "https://raw.githubusercontent.com/Snapchat/snap-kit-carthage/repo/SCSDKBitmojiKit/1.3.5/SCSDKBitmojiKit-1.3.5.zip", "1.3.4": "https://raw.githubusercontent.com/Snapchat/snap-kit-carthage/repo/SCSDKBitmojiKit/1.3.4/SCSDKBitmojiKit-1.3.4.zip"}
```
[the remark this answers is not in the document]
Comment (kemooo3333) [reply SCSDKBitmojiKit.json:1]: ‏rsh-43

[the remark this answers is not in the document]
Comment (kemooo3333) [reply SCSDKBitmojiKit.json:1]: Ok

Comment (kemooo3333) [inline SCSDKBitmojiKit.json:1]: ‏rsh-43

**Review by kemooo3333 — COMMENTED** [review]
‏rsh-43

**Review by kemooo3333 — COMMENTED** [review]
‏rsh-43

File: SCSDKBitmojiKit.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (kemooo3333) [reply SCSDKBitmojiKit.json:1]: ‏rsh-43

[the remark this answers is not in the document]
Comment (kemooo3333) [reply SCSDKBitmojiKit.json:1]: > ‏rsh-43

No

**Review by kemooo3333 — COMMENTED** [review]
‏rsh-43

**Review by kemooo3333 — COMMENTED** [review]
‏rsh-43

**Pull request merged**